src/ResponseExtensions.cs

```diff
@@ -0,0 +1,21 @@
+namespace Carter.HtmlNegotiator
+{
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Carter;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Net.Http.Headers;
+
+    public static class ResponseExtensions
+    {
+        public static async Task AsHtml(this HttpResponse response, string view, object obj, CancellationToken cancellationToken = default)
+        {
```

Comment:
Instead of passing the view name explicitly, it would be good to have some sensible defaults based on the route structure (somehow). As it stands, content negotiation is essentially impossible

Reply:
Yeah, I'm open to ideas here. At the moment I am using the ViewModel name if no view name has been passed in.

https://github.com/CarterCommunity/Carter.HtmlNegotiator/pull/2/files#diff-62bb90acb6596f2418415db11902ef68R57

Reply:
I've been wondering if named routes are a way to go or not, can't make my mind up and it's 11:05pm.

If we did add named routes in some fashion, it would help close off https://github.com/CarterCommunity/Carter/issues/79 as well